### ISSUE TITLE
disableTaggedUnions option ignored in maven plugin

### ISSUE THREAD

**Issue by Q-Man** [vojtechhabarta/typescript-generator#190]
`<disableTaggedUnions>true</disableTaggedUnions>`

But tagged unions are created anyway.

Tested with v1.29.355

**Comment by vojtechhabarta** [comment]
This option disables type aliases with `Union` suffix. With this setting do you still have these types generated? Or could you paste some example what is generated?

**Comment by Q-Man** [comment]
This is the annotation for Jackson:
```
@JsonTypeInfo(use=JsonTypeInfo.Id.CLASS, include=JsonTypeInfo.As.PROPERTY, 
property="@class")
```
This property is generated in the typescript interface:
`"@class": "Company" | "ComplexCompany" | "ConsultantCompany";`

With `disableTaggedUnions=true` I would expect the behaviour from before v1.11.232:
`"@class": string;`

**Comment by vojtechhabarta** [comment]
I fixed your other issue with tagged unions (#189).

As I said `disableTaggedUnions` currently only disables generation of type aliases with `Union` suffix, it doesn't disable union types of discriminant properties. Is this reasonable for you when #189 is fixed now?

**Comment by Q-Man** [comment]
Thanks for the fast reaction, the #189 fix solved my problem. Great tool, by the way. 👍
